doc/whats_new.rst

```diff
@@ -105,6 +105,18 @@ Enhancements
    - Added ``inverse_transform`` function to :class:`decomposition.nmf` to compute
      data matrix of original shape. By `Anish Shah`_.
 
+    - Enhanced MLP with Adams, Nesterov's momentum, and early stopping
+      (`#5214 <https://github.com/scikit-learn/scikit-learn/pull/5214>`_)
```

Comment:
this PR was started by Jiyuan Qian, https://github.com/glennq this was a huge PR, is that why there is no particular name attached to this? Otherwise I suggest to add it